src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -41,16 +42,26 @@ Copyright (c) .NET Foundation. All rights reserved.
     Remove @(CustomAdditionalCompileInputs) and @(CustomAdditionalCompileOutputs)
     Remove @(DocFileItem)
     Remove PdbFile="$(PdbFile)"
+    Remove $(IntermediateRefAssembly)
     Remove OutputRefAssembly="@(IntermediateRefAssembly)"
     Remove MainEntryPoint="$(StartupObject)"
+    Remove winmdobj <PdbFile  ....> and comment
+    Remove RefOnly="$(ProduceOnlyReferenceAssembly)"
+    Remove Win32Icon="$(ApplicationIcon)"
+    Remove Win32Manifest="$(Win32Manifest)"
+    Remove Win32Resource="$(Win32Resource)"
 
     Remove EmbedAllSources="$(EmbedAllSources)" - not supported by our supported version of MSBuild
 
     Remove additional steps after calling CSC
 
     Add our FileWrites after the call to CSC
+    
+    Add Condition="'@(RazorCompile)'!=''" as a condition to the RazorCoreCompile target.
   -->
- <Target Name="RazorCoreCompile"
+  <Import Project="Microsoft.Managed.Core.targets"/>
```

Comment:
This isn't available, so I technically should remove it or copy it all. Thoughts?

Reply:
Going to nuke it unless told otherwise

Reply:
I dunno what that is?

Reply:
It's all super arcane to me: https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

Reply:
I think we should take it out since it's an import. It will already be included elsewhere.